test/metabase/lib/order_by_test.cljc

```diff
@@ -87,3 +92,46 @@
           (-> (lib/query-for-table-name meta/metadata-provider "VENUES")
               (lib/order-by (lib/field "VENUES" "ID"))
               lib/order-bys))))
+
+;;; the following tests use raw legacy MBQL because they're direct ports of JavaScript tests from MLv1 and I wanted to
+;;; make sure that given an existing query, the expected description was generated correctly.
+
+(defn- describe-legacy-query-order-by [query]
+  (-> (lib.query/query meta/metadata-provider (lib.convert/->pMBQL query))
+      (lib.metadata.calculation/describe-top-level-key -1 :order-by)))
+
+(deftest ^:parallel describe-order-by-test
+  (let [query {:database (meta/id)
+               :type     :query
+               :query    {:source-table (meta/id :venues)
+                          :order-by     [[:asc [:field (meta/id :venues :category-id) nil]]]}}]
+    (is (= "Sorted by Category ID ascending"
+           (describe-legacy-query-order-by query)))))
+
+(deftest ^:parallel describe-order-by-aggregation-reference-test
+  (let [query {:database (meta/id)
+               :type     :query
+               :query    {:source-table (meta/id :venues)
+                          :aggregation  [[:count]]
+                          :breakout     [[:field (meta/id :venues :category-id) nil]]
+                          :order-by     [[:asc [:aggregation 0]]]}}]
+    (is (= "Sorted by Count ascending"
+           (describe-legacy-query-order-by query)))))
+
+(deftest ^:parallel describe-order-by-expression-reference-test
+    ;;   it("should work with expressions", () => {
```

Comment:
looks like an unimplemented test

Reply:
Oops. I'll fix this in the follow on PR I'm working on that actually flips the switch to use this in JavaScript, I need to add a few more tests there anyway